src/title/Title.unit.tsx

```diff
@@ -1,38 +1,60 @@
 import React from 'react'
-import { shallow } from 'enzyme'
 
-import { Title } from './Title'
+import { render, screen } from '@testing-library/react'
+
+import { Title, TitleProps } from './Title'
+
+const defaultProps: TitleProps = {
+  children: 'Hello',
+  headingLevel: 1,
+}
+
+function createProps(props: Partial<TitleProps> = {}): TitleProps {
+  return { ...defaultProps, ...props }
+}
 
 describe('Title', () => {
   it('Should render the given title text', () => {
     const title = 'blabla'
-    const wrapper = shallow(<Title>{title}</Title>)
-    expect(wrapper.hasClass('kirk-title')).toBe(true)
-    expect(wrapper.text()).toContain(title)
-    expect(wrapper.name()).toBe('h1')
+    const props = createProps({
+      children: title,
+    })
+
+    render(<Title {...props} />)
+
+    expect(screen.getByRole('heading')).toHaveTextContent(title)
```

Comment:
nit: You can use `props.title` instead of `title` and remove the local variable 👌.